src/SFML.Graphics/RenderTexture.cs

```diff
@@ -266,58 +233,43 @@ public Vector2i MapCoordsToPixel(Vector2f point, View view)
         ///
         /// <returns>True if mipmap generation was successful, false if unsuccessful</returns>
         ////////////////////////////////////////////////////////////
-        public bool GenerateMipmap()
-        {
-            return sfRenderTexture_generateMipmap(CPointer);
-        }
+        public bool GenerateMipmap() => sfRenderTexture_generateMipmap(CPointer);
 
         ////////////////////////////////////////////////////////////
         /// <summary>
         /// Clear the entire render texture with black color
         /// </summary>
         ////////////////////////////////////////////////////////////
-        public void Clear()
-        {
-            sfRenderTexture_clear(CPointer, Color.Black);
-        }
+        public void Clear() => sfRenderTexture_clear(CPointer, Color.Black);
 
         ////////////////////////////////////////////////////////////
         /// <summary>
         /// Clear the entire render texture with a single color
         /// </summary>
         /// <param name="color">Color to use to clear the texture</param>
         ////////////////////////////////////////////////////////////
-        public void Clear(Color color)
-        {
-            sfRenderTexture_clear(CPointer, color);
-        }
+        public void Clear(Color color) => sfRenderTexture_clear(CPointer, color);
 
         ////////////////////////////////////////////////////////////
         /// <summary>
         /// Update the contents of the target texture
         /// </summary>
         ////////////////////////////////////////////////////////////
-        public void Display()
-        {
-            sfRenderTexture_display(CPointer);
-        }
+        public void Display() => sfRenderTexture_display(CPointer);
 
         ////////////////////////////////////////////////////////////
         /// <summary>
         /// Target texture of the render texture
         /// </summary>
         ////////////////////////////////////////////////////////////
-        public Texture Texture
-        {
-            get { return myTexture; }
-        }
+        public Texture Texture { get; } = null;

```

Comment:
Is the `null` init not unnecessary as it's the default value?

Reply:
It's not, I've configured a rule to warn about it in the future.